File: tests/cc/iboard/StateEndpointTest.java
package cc.iboard;

import cc.iboard.backend.Response;
import cc.iboard.backend.TestBackend;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class StateEndpointTest {

    private static final TestBackend backend = new TestBackend();

    @Test
    void testStateResponse() {
        Response response = TestHelper.send(backend,"/state");
        assertTrue(response.body().contains("Running"));
    }

}

File: tests/cc/iboard/StateEndpointTest.kt
package cc.iboard

import cc.iboard.backend.Response
import cc.iboard.backend.TestBackend
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.*

internal class StateEndpointTest {

    @Test
    fun testStateResponse() {
        val response = TestHelper.send(backend, "/state")
        assertTrue(response.body().contains("Running"))
    }

    companion object {

        private val backend = TestBackend()
    }

}
